dash/development/_r_components_generation.py

```diff
@@ -422,7 +422,7 @@ def write_help_file(name, props, description, prefix, rpkg_data):
             funcname=funcname,
             name=name,
             default_argtext=textwrap.fill(default_argtext,
-                                          width=80,
+                                          width=60,
```

Comment:
60, 70, 80? is `60` wrong?

Reply:
`60` is correct, can't change text of commit message easily with rebase given the merge commits in the way -- figured it will get squashed in the end, and disappear. Sorry, should've explained offline.